### ISSUE TITLE
Which value in context.contextActivities.grouping.definition.type?

### ISSUE THREAD

**Issue by wilmTap** [jiscdev/xapi-vle#72]
The VLE Assignment Graded and VLE User Submits Assignment Recipes – and its Blackboard exemplars – has <http://adlnet.gov/expapi/activities/course>, but the Moodle exemplar has <http://xapi.jisc.ac.uk/define/vle/course>

The adl:course is defined as:

> A course represents an entire “content package” worth of material. The largest level of granularity. Unless flat, a course consists of multiple modules.

That seems to me to be at odds with the sort of granularity of "course" in the UDD or HESA sense: a degree programme. Moreover, the link between any VLE courseArea and a module is already a bit loose (it's disambiguated by <http://xapi.jisc.ac.uk/extensions/vle_mod_id>), so I'm not sure that there would be anything reliable in a typical VLE above the level of module.

In short, should the value be: <http://adlnet.gov/expapi/activities/module>?

I think this has a bearing on issue #65

**Comment by arc12** [comment]
WIlbert, all -
in general I am against using terms with semantics of content aggregation, which applies to both the adl course and module definitions. Part of this is an objection to the equation that online-stuff = course, but it seems at odds with user-created stuff,
On the other hand, the association with a UDD-sense course is indirect, imprecise, and potentially unknown at statement-creation time.

I think it really would be safest to define something distinct and to note the possible equivalence to adl defined terms in some circumstances/systems (e.g. fully on-line self-paced independent study). I think this distinct thing is describable as a "course site" (although I've used "course area" before, and I can get the gist of compounds like "vle-course"), and its got to have VLE-land semantics IMO. Maybe it can be associated with a UDD course, or be found to be equivalent to an adl:course but its not bound to be ...

So, my vote is: coin afresh, reconcile later if necessary. Otherwise we're essentially saying that the adlnet definition should be ignored (or re-written, which I think is not in remit).

Adam

**Comment by garemoko** [comment]
+1 @arc12 The adl course and module terms are designed for and used by e-learning content. If we use them for VLE courses/modules then a VLE course/module contains e-learning courses/modules it will get confusing to use the same identifiers.

**Comment by wilmTap** [comment]
That's clear, and I agree that the VLE structure should be clearly separate from the UDD structure. However, I can't really see a sensible level in the VLE structure above that of the courseArea in most cases- and maybe that's how it should be?
